lld/test/COFF/reloc-undefined-weak.s

```diff
@@ -0,0 +1,52 @@
+// REQUIRES: x86
+
+// Check that base-relocations for unresolved weak symbols will be omitted.
+
+// RUN: rm -rf %t.dir && split-file %s %t.dir && cd %t.dir
+// RUN: llvm-mc -filetype=obj -triple=x86_64-mingw main.s -o main.o
+// RUN: llvm-mc -filetype=obj -triple=x86_64-mingw other.s -o other.o
+
+// RUN: ld.lld -m i386pep -dll -o other.dll other.o -entry= --export-all-symbols --out-implib other.dll.a
```

Comment:
We generally don't use the `ld.lld` interface in `lld/test/COFF`, but only use the direct `lld-link` interface - even if it understandably would make more sense to use the GNU ld-like interface for being able to crosscheck things with GNU ld.